lib/Parser/pnodewalk.h

```diff
@@ -174,10 +174,19 @@ class ParseNodeWalker : public WalkerPolicy
     {
         ResultType result;
         // For ordering, arguments are considered prior to the function and the body after.
-        for (ParseNodePtr* argNode = &(pnode->AsParseNodeFnc()->pnodeParams); *argNode != nullptr; argNode = &((*argNode)->AsParseNodeVar()->pnodeNext))
```

Comment:
>&((*argNode)->AsParseNodeVar()->pnodeNext [](start = 109, length = 41)

Here we had a bad cast, but because `pnodeNext` is at the same offset for ParseNodeVar and ParseNodeParamPattern, the walker was fine